pkg/ctl/cmdutils/configfile_test.go

```diff
@@ -301,6 +303,79 @@ var _ = Describe("cmdutils configfile", func() {
 	})
 })
 
+type scaleNodeGroupCase struct {
+	include []string
+	exclude []string
+	error   error
+}
+
+var _ = Describe("scale node group config file loader", func() {
+	newCmd := func() *cobra.Command {
+		return &cobra.Command{
+			Use: "test",
+			Run: func(_ *cobra.Command, _ []string) {},
+		}
+	}
+
+	DescribeTable("create nodegroup successfully",
+		func(params scaleNodeGroupCase) {
+			cmd := &Cmd{
+				CobraCommand:      newCmd(),
+				ClusterConfigFile: "test_data/scale-ng-test.yaml",
+				ClusterConfig:     api.NewClusterConfig(),
+				ProviderConfig:    &api.ProviderConfig{},
+				Include:           params.include,
+				Exclude:           params.exclude,
+			}
+
+			ngFilter := NewNodeGroupFilter()
+			ng := &api.NodeGroup{}
+			if err := NewScaleNodeGroupLoader(cmd, ng, ngFilter).Load(); err != nil {
+				Expect(err.Error()).To(Equal(params.error.Error()))
+			} else {
+				Expect(params.error).NotTo(HaveOccurred())
+				Expect(ng.DesiredCapacity).To(Not(Equal(0)))
+			}
```

Comment:
This needs to be written in a different way so that in case of error the messages make sense and don't panic (the argument of `Expect()` should always be the actual value):

```
err := NewScaleNodeGroupLoader(cmd, ng, ngFilter).Load()
if params.error != nil {
    Expect(err).To(HaveOccurred())
	Expect(err.Error()).To(ContainSubstring(params.error.Error()))
} else {
	Expect(err).ToNot(HaveOccurred())
}
```